
syntax = "proto3";
package c_ci;
// The greeting service definition.
service Greeter {
// Sends a greeting
rpc CallTest (TestRequest)returns (TestReply){}
}

// The request message containing the user's name.
message TestRequest {
string path= 1;
}
// python3 -m grpc_tools.protoc -I./ --python_out=. --pyi_out=. --grpc_python_out=. ./ci.proto
// python3 -m grpc_tools.protoc -I. --python_out=. --pyi_out=. --grpc_python_out=. ./ci.proto
message TestReply {
    string retMsg=1;
}
